test/Base_Tests/src/Semantic/Multi_Value_Spec.enso

```diff
@@ -14,6 +35,61 @@ add_specs suite_builder =
             (c:Complex).re . should_equal 1.5
             (c:Float) . should_equal 1.5
 
+    suite_builder.group "Chain Multi Value" group_builder->
+        to_b_to_c obj =
+            b = obj : B
+            c = b : C
+            c
+
+        group_builder.specify "Multiple conversions of A work" <|
+            a = A
+            a_text = to_b_to_c a
+            a_text . to_text . should_equal "(C (B A))"
```

Comment:
I know you really don't like comments about variable naming, but it is sometimes misleading.
Looking at the context here, `a_text` is not really a Text, perhaps it was in earlier iterations of this test? Can we call it something less misleading? E.g. `a_converted`?

Reply:
Done in 57eaf4acd6